src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/ManagedCase.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.ManagedOffenderCrn
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.probation.OffenderDetail
+
+@Schema(description = "Describes an CA(OMU) caseload")
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏If these classes are only used in the creation of the caseload, better in include them as classes within the caseload service itself?

Reply:
This is shared for COM caseload as well.

Reply:
**question:** is this exposed via the API? 
(It should be in the service layer only if not)